tools/accuracy_checker/accuracy_checker/adapters/pose_estimation.py

```diff
@@ -101,25 +105,33 @@ def process(self, raw, identifiers, frame_meta):
             pad = meta.get('padding', [0, 0, 0, 0])
             transpose_order = (1, 2, 0) if heatmap.shape[0] == 19 else (0, 1, 2)
 
+            s = 8
```

Comment:
what the reason of this changes? you just use named variable instead of numeric value (to be honest, the code is not become more readable with this change)

Reply:
The reason is that it's better to keep it in a separate variable, when you try different values for it. This is a debug code, though. Removed.